src/mp-instance.js

```diff
@@ -1292,189 +1295,68 @@ function completeSDKInitialization(apiKey, config, mpInstance) {
         );
-
         mpInstance._Forwarders.processPixelConfigs(config);
 
+        // Checks if session is created or resumed
+        // Logs a session start event
         mpInstance._SessionManager.initialize();
         mpInstance._Events.logAST();
 
-        // Call mParticle._Store.SDKConfig.identityCallback when identify was not called due to a reload or a sessionId already existing
-        // Any identity callback should always be ran regardless if an identity call is made
-        if (
-            !mpInstance._Store.identifyCalled &&
-            mpInstance._Store.SDKConfig.identityCallback &&
-            currentUser &&
-            currentUser.getMPID()
-        ) {
-            mpInstance._Store.SDKConfig.identityCallback({
-                httpCode: HTTPCodes.activeSession,
-                getUser: function() {
-                    return mpInstance._Identity.mParticleUser(
-                        currentUser.getMPID()
-                    );
-                },
-                getPreviousUser: function() {
-                    var users = mpInstance.Identity.getUsers();
-                    var mostRecentUser = users.shift();
-                    if (
-                        mostRecentUser &&
-                        currentUser &&
-                        mostRecentUser.getMPID() === currentUser.getMPID()
-                    ) {
-                        mostRecentUser = users.shift();
-                    }
-                    return mostRecentUser || null;
-                },
-                body: {
-                    mpid: currentUser.getMPID(),
-                    is_logged_in: mpInstance._Store.isLoggedIn,
-                    matched_identities: currentUser.getUserIdentities()
-                        .userIdentities,
-                    context: null,
-                    is_ephemeral: false,
-                },
-            });
-        }
+        processIdentityCallback(
+            mpInstance,
+            currentUser,
+            currentUserMPID,
+            currentUserIdentities
+        );
     }
 
     mpInstance._Store.isInitialized = true;
-    // Call any functions that are waiting for the library to be initialized
-    if (
-        mpInstance._preInit.readyQueue &&
-        mpInstance._preInit.readyQueue.length
-    ) {
-        mpInstance._preInit.readyQueue.forEach(function(readyQueueItem) {
-            if (mpInstance._Helpers.Validators.isFunction(readyQueueItem)) {
-                readyQueueItem();
-            } else if (Array.isArray(readyQueueItem)) {
-                processPreloadedItem(readyQueueItem, mpInstance);
-            }
-        });
 
-        mpInstance._preInit.readyQueue = [];
+    // Call any functions that are waiting for the library to be initialized
+    try {
+        mpInstance._preInit.readyQueue = processReadyQueue(
+            mpInstance._preInit.readyQueue
+        );
+    } catch (error) {
+        mpInstance.logger.verbose(error);
```

Comment:
I knwo it was `verbose` further down the chain, but I think it should be `error`.  This is an innocuous refactor and I'm okay with making the change here.
```suggestion
        mpInstance.logger.verbose(error);
```